### PR TITLE
Added 90% ES CPU alarms to those disabled for the data refresh

### LINKED ISSUE #4826: Add 90% ES CPU alarms to the list of those disabled for the data refresh
## Description

Presently, we disable the Elasticsearch 50% CPU alarm during the course of the data refresh process. Since #4683, the data refresh has also been tripping the Elasticsearch 90% CPU alarm as well. We should add this to the list of alarms which get disabled. Specifically:

https://github.com/WordPress/openverse/blob/d500b7764c411f7d228ae12c57dce519c8709610/catalog/dags/common/cloudwatch.py#L73-L72

We should add another `enable_alarm_actions` call for `ES Prod CPU utilization above 90%`.

### PR DESCRIPTION
**Pull request by coson-lu**
…freshed (Issue #4826)

## Fixes

Fixes #4826 by @AetherUnbound

## Description
This pull requests adds the Elasticsearch 90% CPU alarm to the list of alarms which get disabled for the refresh data.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
- [ ] I ran the DAG documentation generator (`ov just catalog/generate-docs` for catalog
      PRs) or the media properties generator (`ov just catalog/generate-docs media-props`
      for the catalog or `ov just api/generate-docs` for the API) where applicable.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by stacimc — APPROVED** [review]
Perfect, thanks @coson-lu!

File: catalog/dags/common/cloudwatch.py [hunk 1]
```diff
@@ -71,3 +71,4 @@ def enable_or_disable_alarms(enable):
     )
     cw_wrapper = CloudWatchWrapper(cloudwatch.get_conn())
     cw_wrapper.enable_alarm_actions("ES Production CPU utilization above 50%", enable)
+    cw_wrapper.enable_alarm_actions("ES Production CPU utilization above 90%", enable)
```
Comment (AetherUnbound) [inline catalog/dags/common/cloudwatch.py:74]: Sorry for the slight correction, this alarm is actually _slightly_ different (I'll make an issue to correct this in our infrastructure repository):
```suggestion
    cw_wrapper.enable_alarm_actions("ES Prod CPU utilization above 90%", enable)
```

**Review by AetherUnbound — CHANGES_REQUESTED** [review]
One small change which I'll make in-line, just didn't want this to get merged first

**Review by AetherUnbound — APPROVED** [review]
Sweet, thanks for the contribution @coson-lu!

**Comment by sarayourfriend** [comment]
Just a heads-up @AetherUnbound and any other spectators of this issue: modifying alarm states from DAGs requires IAM changes to give the Airflow execution environment permissions to do so. I've gone ahead and fixed it for this alarm, but keep that in mind when interacting with AWS resources in the future. Access is enabled and controlled via IAM and permissions errors will happen in the live environment without those additional changes.

**Comment by AetherUnbound** [comment]
Ahh, that was a complete oversight on my part. Thanks for the notice 🙂

**Pull request merged**